enterprise_subsidy/apps/transaction/management/commands/backpopulate_transaction_email_and_title.py

```diff
@@ -0,0 +1,65 @@
+"""
+Management command to backpopulate transaction email and title
+"""
+import logging
+
+from django.core.management.base import BaseCommand
+from django.db.models import Q
+from openedx_ledger.models import Transaction
+
+from enterprise_subsidy.apps.subsidy.models import Subsidy
+from enterprise_subsidy.apps.transaction.utils import batch_by_pk
+
+logger = logging.getLogger(__name__)
+
+
+class Command(BaseCommand):
+    """
+    Management command for backpopulating transaction email and title
+
+    ./manage.py backpopulate_transaction_email_and_title
+    """
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.dry_run = False
+
+    def add_arguments(self, parser):
+        """
+        Entry point for subclassed commands to add custom arguments.
+        """
+        parser.add_argument(
+            '--dry-run',
+            action='store_true',
+            dest='dry_run',
+            help=(
+                'If set, no updates will occur; will instead log '
+                'the actions that would have been taken.'
+            ),
+        )
+
+    def handle(self, *args, **options):
+        """
+        Send it
+        """
+        if options.get('dry_run'):
+            self.dry_run = True
+            logger.info("Running in dry-run mode. No updates will occur.")
+
+        incomplete_only_filter = Q(lms_user_email__isnull=True) | Q(content_title__isnull=True)
+        for items_batch in batch_by_pk(Transaction, extra_filter=incomplete_only_filter):
```

Comment:
Nit: `batch_by_pk` is really cool. OTOH, you could loop over Subsidies first to do some level of batching for you, and then the sequence of work would have some level of predictability, and you'd only need to fetch each subsidy record once.  It also opens the door for us to do bulk fetches (filtered by the related subsidy record's enterprise uuid) in the future.